common/services/person.test.js

```diff
@@ -4,9 +4,10 @@ const apiClient = require('../lib/api-client')()
 
 const unformatStub = sinon.stub()
 
-const personService = proxyquire('./person', {
+const PersonService = proxyquire('./person', {
   './person/person.unformat': unformatStub,
 })
+const personService = new PersonService(apiClient)
```

Comment:
The service constructor takes a req object, so I think this should be ({ apiClient }) or something like that to mock a req object with the apiClient attached.

Reply:
Good catch :)